doc/Gabc.tex

```diff
@@ -109,10 +113,58 @@ \subsection{Headers}
 \item[manuscript-storage-place] For transcriptions direct from a manuscript, where the manuscript is held; e.g. Bibliothèque Nationale, Paris.
 \item[book] For transcriptions from a modern book (such as Solesmes editions; modern goes back at least to the 19th century revival), the name of the book; e.g. Liber Usualis.
 \item[transcriber] The name of the transcriber into gabc.
-\item[transcription-date] The date the gabc was written, with the following convention yyyymmdd, like 20090129 for january the 29th 2009.
+\item[transcription-date] The date the gabc was written, with the following convention yyyymmdd, like 20090129 for January the 29th 2009.
 \item[user-notes] This may contain any text in addition to the other headers -- any notes the transcriber may wish. However, it is recommended to use the specific header fields where they are suitable, so that it is easier to find particular information.
 \end{description}
 
+\subsubsection{Mode Headers}
+
+The three mode headers described above (\texttt{mode}, \texttt{mode-modifier},
+and \texttt{mode-differentia}) will be typeset above the initial if neither
+the \texttt{annotation} gabc header nor the \verb=\greannotation= \TeX{}
+command is used.
+
+The mode annotation will look like
+\writemode{mode}{\thinspace mode-modifier}{\thinspace mode-differentia}.
+
+The \texttt{mode} header is typically a number that will be typeset as a
+Roman numeral using the \texttt{modeline} style.  However, there are other
+modes.  If the first character of \texttt{mode} is a number from one
+(\texttt{1}) through eight (\texttt{8}), that number will be converted
```

Comment:
The sentence which starts with "However" seems out of place to me, but moving it has some knock on effects to the rest of the paragraph.  I might try it as follows:

> The `mode` header is typically a number that is typeset as a Roman numeral using the `modeline` style.  Therefore, if the first character of `mode` is a number from one (`1`) through eight (`8`), that number will be converted according to the `\gresetmodenumbersystem` setting.  However, there are other modes, so all other values for `mode` will be typeset directly.  If the `mode` header is omitted, none of the other headers will be typeset.